jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: classic-docs
+title: "API v2 Introduction"
+short-title: "Introduction"
+description: "introduction to the CircleCI API v2"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI API v2 is a new API that may be used by internal and external users to make API calls to retrieve detailed information about workflows and pipelines.
+
+## Introduction to API v2
+
+CircleCI API v2 enables you to use a brand-new API with a set of endpoints and several new features that improve the API experience, in addition to optimizing how you use the API for your jobs. Some of these features include the ability to:
+
+- Retrieve a workflow by ID
+- Trigger pipelines with parameters
+- Trigger particular workflows within pipelines
+- Retrieve a pipeline by ID
+- Retrieve recent pipelines for an organization or for a project (ideally with branch filtering)
+
+## Getting Started with the API v2
+
+The v2 API is very similar to the current v1.1 API with some notable exceptions.
```

Comment:
I'd suggest just diving right into it. Since we're not even doing a compare/contrast, we should start right off with explaining `${project_slug}`

Reply:
fixed